smarc_stonefish_worlds/launch/stonefish.launch

```diff
@@ -22,6 +23,9 @@
     <param name="utm_zone" type="int" value="$(arg utm_zone)"/>
     <param name="utm_band" type="string" value="$(arg utm_band)"/> 
 
+    <!-- Put world_config_file params in global namespace -->
+    <rosparam file="$(arg world_config_file)" subst_value="true"/>
```

Comment:
Does it need to be in the global namespace? You will still have access to them in `/stonefish_simulator/[your param]`

Reply:
Hmm... I was thinking that we want to access `/marked_positions` regardless in simulation or in real life...?

Reply:
I'm concerned that this pollutes the global namespace with parameters that might be confusing. E.g. `/environment_file`, `/latitude` or `/longitude` might suggest that they have some particular meaning, whereas in reality, they are just implementation parameters of our stonefish simulation environment.

Reply:
Hmm... That is a good point... I'll remove it, sorry for the messy commits :see_no_evil: